components/brave_rewards/resources/page/components/sidebar_promotion_panel.tsx

```diff
@@ -15,39 +15,43 @@ import { CloseIcon } from '../../shared/components/icons/close_icon'
 
 import * as style from './sidebar_promotion_panel.style'
 
-interface PromotionMessages {
+interface PromotionMessage {
   title: string
-  text: string
-  disclaimer: string
+  text1: string
+  text2?: string
+  disclaimer?: string
 }
 
-function getPromotionMessages (
+function getPromotionMessage (
```

Comment:
Nit: the plural/singular thing can be odd in a case like this. I would probably just leave it `PromotionMessages`/`getPromotionMessages`.

Reply:
Sure, I reverted that change.